src/main/webapp/angular.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
fyi, I don't know Angular so will not be able to give in-depth comments on these